build.py

```diff
@@ -169,22 +161,19 @@ def xcodebuild_command(self, operation, xcpretty) :
 		
 		# The shallow analyzer is buggy. Stupidly buggy, causing random failures that didn't fail the build on things like
 		# headers not being found. If Apple can't make this reliable then we should short circuit it out of our build
-		
 		if (operation == "build") :
-			command += " RUN_CLANG_STATIC_ANALYZER=NO ONLY_ACTIVE_ARCH=YES VALID_ARCHS=x86_64"
+			command += " RUN_CLANG_STATIC_ANALYZER=NO"
 		
 		if (operation != None and "codecov" in self.operations) :
 			command += " -enableCodeCoverage YES"
 
 		if (self.platform == "iOS") :
-			command += " " + ios_sim_flags + " " + ios_sim_dest + " SUPPORTED_PLATFORMS.DEBUG=iOS"
+			command += " " + ios_sim_flags + " " + ios_sim_dest
```

Comment:
Here also why did we remove SUPPORTED_PLATFORMS.DEBUG=iOS

Reply:
This param isn't needed. The script was copied from the MSAL repo, we use the same logic there.